frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts

```diff
@@ -0,0 +1,67 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { NooBaaObjectBucketClaimModel } from 'packages/noobaa-storage-plugin/src/models';
+import {
+  storageClassDropDown,
+  defaultSC,
+  scDropdown,
+  createBtn,
+} from '../views/bucketClaimCreation.view';
+import {
+  obcCount,
+  clusterHealth,
+  efficiencyValue,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const BROWSER_TIMEOUT = 30000;
+
+describe('Test Object Service Dashboard Page', () => {
```

Comment:
Its better to move ```await browser.get(`${appHost}/dashboards/object-service`);``` and the visibility check to `beforeEach` block to remove duplication.